Test/Framework/Visual/VideoTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Media;
+using NUnit.Framework;
+using System;
+
+namespace MonoGame.Tests.Visual
+{
+    [TestFixture]
+    class VideoTests : VisualTestFixtureBase
+    {
+        SpriteBatch _spriteBatch;
+        Video _video;
+        VideoPlayer _player;
+        TimeSpan _start;
+
+        [SetUp]
+        public override void SetUp()
+        {
+            base.SetUp();
+
+            Game.LoadContentWith += (sender, e) =>
+            {
+                _spriteBatch = new SpriteBatch(Game.GraphicsDevice);
+                _video = Game.Content.Load<Video>(Paths.Video("SampleVideo_360x240_5s"));
+                _player.Play(_video);
```

Comment:
@KonajuGames Can we take another look at getting this resolved? _player is just not initialized here. 

Not necessarily for this PR, but these tests could be improved by saving some reference frames for the videos. That way this does not need to actually run a game and can use the GraphicsDeviceTestFixtureBase instead of the VisualTestFixtureBase. Checking the frames makes the tests more robust too.